src/isotools/splice_graph.py

```diff
@@ -397,7 +399,7 @@ def _check_exon(self, j1, j2, is_first, is_reverse, e, e2=None):
                 if troi:
```

Comment:
`troi` and `ji` only occur between L392 and L408 in this script. Maybe it would help to understand the code if explain them, if you know what they are short for?